app/views/users/index.html.erb

```diff
@@ -1,10 +1,13 @@
-<h1>User Data</h1>
 
 <ul>
   <% @user_data.each do |user| %>
```

Comment:
- [ ] Kindly consider Using #recent_post method to render the user's 3 most recent post to meet the project requirements

Reply:
well resolved! ✔️